stage1/init/init.go

```diff
@@ -369,6 +551,14 @@ func stage1() int {
 		return 3
 	}
 
+	appHashes := p.GetAppHashes()
+	s1Root := common.Stage1RootfsPath(p.Root)
+	machineID := p.GetMachineID()
+	if err := createCgroups(s1Root, machineID, appHashes); err != nil {
```

Comment:
Actually, they work on v219 because systemd-nspawn leaves the cgroups mounted RW :+1: